### PR TITLE
follow naming convention

### PR DESCRIPTION
**Pull request by mitmarcus**

### CODE REVIEW

File: lib/encryption/cryptorator.c [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+#include <time.h>
+#include <stdint.h>
+#include <string.h>
+
+#include "aes.h"
+
+static uint8_t key[16];
+
+void cryptorator_init(uint8_t *key){
+    for (int i = 0; i < 16; i++) {
```
Comment (JakubCerovsky) [inline lib/encryption/cryptorator.c]: Please create a specific branch for this file so it wont be messed up

**Review by JakubCerovsky — CHANGES_REQUESTED** [review]
Like the refactor but the cryptorator file is a feature not a refactor thingy.

**Comment by mitmarcus** [comment]
ah that's very annoying, idk how just that one specific file got in this branch

**Review by mitmarcus — COMMENTED** [review]
all good

**Review by JakubCerovsky — COMMENTED** [review]
Nice Job!

**Pull request merged**